.github/ISSUE_TEMPLATE/10_bug_report.yml

```
@@ -8,6 +8,11 @@ body:
       value: |
         Thanks for taking the time to fill out this bug report correctly.
 
+        Please report only bug related to QGIS Desktop or Server.
```

```suggestion
        Please report only issues related to QGIS.
```

... there's also qgis_process, not sure we want to go into that much detail here